docs/001_develop/02_server-capabilities/003_data-access-apis/index.mdx

```diff
@@ -179,7 +178,7 @@ All three entity DB APIs (`SyncEntityDb`, `AsyncEntityDb`, `RxEntityDb`) use the
 
 #### `get`
 
-Get is a simple lookup on the database; it will return a single entity if a match is found, or no records if none is
+Get is a simple look-up on the database; it returns a single entity if a match is found, or no records if no match is found.
 found.
```

Comment:
@wjhendry orphaned 'found' word on this line

Reply:
Thanks.